packages/api/cms-api/src/blocks/PixelImageBlock.ts

```diff
@@ -54,7 +56,7 @@ class PixelImageBlockData extends BlockData {
                           dominantColor: file.image.dominantColor,
                       }
                     : undefined,
-                fileUrl: await filesService.createFileUrl(file, previewDamUrls),
+                fileUrl,
```

Comment:
Should we make fileUrl nullable in the block meta?

Reply:
I thought about that; in admin it's not nullable, in site it's always null.

for attachedBlocks of OneOfBlock (which is also dependent on includeInvisibleContent) we also don't use nullable.

so basically an edge case we can't cover with out meta system